cypress/integration/app.spec.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types="cypress" />
+
+describe('Homepage', () => {
+  it('displays a welcome text', () => {
+    cy.visit('/');
+    cy.contains('Welcome');
```

Comment:
Although it's not always the [best](https://docs.cypress.io/guides/references/best-practices#Text-Content) practice to use `contains`, used it as this test only for setup purpose!